src/controller/scan/base_controller.go

```diff
@@ -559,9 +559,10 @@ func (bc *basicController) GetReport(ctx context.Context, artifact *ar.Artifact,
 	groupReports := make([][]*scan.Report, len(artifacts))
 
 	var wg sync.WaitGroup
+	var artifactIDs []int64
 	for i, a := range artifacts {
 		wg.Add(1)
-
+		artifactIDs = append(artifactIDs, a.ID)
```

Comment:
Looks like this statement doesn't have an effect.  Remove it?

Reply:
Updated